### DISCUSSION TITLE
After ^9.4.0 static images are delivering extermely slowly

### DISCUSSION THREAD

**Discussion by sambhav-gore** [vercel/next.js#12932] [Help] [answered]
I posted it as comment in other discussion but this issue is much different than that so I am creating  a new one here.

The issues is that I have some Static images in the /public folder between ~100kb to ~300kb file size each. Just after the upgrade to 9.4.0 it has become Extremely slow to come from the server. The rest of the things at lightning fast speed but something terribly wrong with images. Has anyone else seen this issue?
I am not on now but I do not think it is a platform issue.

**Comment by sambhav-gore** [comment]
I tried `next@9.4.1-canary.5 ` and it is still the same

**Comment by ijjk** [comment, marked as answer]
Hi, can you provide a repo with a reproduction where this is occurring in the latest version but not in a previous version? Have any other factors in your environment where you are seeing this behavior changed besides the Next.js version?

**Reply by sambhav-gore** [reply]
Hi, I created two deployments with different versions on Now and they are working identical (very fast). 
So i think it is just a coincidence that my server has become slower to respond when I also upgraded to latest next.

for now I have gotten around the issue by moving the static images to a cdn. sorry for the trouble.and thanks for the quick response.

**Reply by Jagadesh-Ram** [reply]
What about serving the images from a database like firebase??? Am using this method currently but the fetched images is slowly loaded.
Any workaround??

**Reply by sambhav-gore** [reply]
I moved it over to a CDN and the performance is very good. I am not sure if serving static images from DB is the best idea.

**Reply by Jagadesh-Ram** [reply]
Oh okay, then what is the use of firebase database then? and if any change made to the database will it asynchronously affect the cdn???
also which CDN is recommended for the best image  and gif serving when using NextJS?
